pkg/api/mlflow/service/artifact/storage/local_test.go

```diff
@@ -7,8 +7,83 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config"
 )
 
+// TestGetArtifact for the local storage implementation.
+func TestGetArtifact_Ok(t *testing.T) {
+	// setup
+	defaultArtifactRoot := t.TempDir()
+	runArtifactRoot := "/run-artifact-root/"
+	fileName := "file.txt"
+	fileContent := "artifact content"
+
+	err := os.MkdirAll(filepath.Join(defaultArtifactRoot, runArtifactRoot), os.ModePerm)
+	assert.Nil(t, err)
+
+	f, err := os.Create(filepath.Join(defaultArtifactRoot, runArtifactRoot, fileName))
+	assert.Nil(t, err)
+	_, err = f.Write([]byte(fileContent))
+	assert.Nil(t, err)
+
+	// invoke
+	svcConfig := &config.ServiceConfig{
+		DefaultArtifactRoot: defaultArtifactRoot,
+	}
+	svc, err := NewLocal(svcConfig)
```

Comment:
sorry, it isn't a service, it is a `storage`. can you please align names? the same probably about `svcConfig` could be just `config` or `cfg`.

Reply:
can change but i think the term "service" can apply generally to things which provide a functionality. The config is actually an instance of ServiceConfig so why rename?